### DISCUSSION TITLE
1.3.2 Release

### DISCUSSION THREAD

**Discussion by jertel** [jertel/vuegraf#61] [General]
- Add support for nested devices

Known Issue:
- The detailed per-second data is not being adjusted correctly by Emporia's servers for nested devices.

<hr /><em>This discussion was created from the release <a href='https://github.com/jertel/vuegraf/releases/tag/1.3.2'>1.3.2</a>.</em>

**Comment by LaughLax** [comment]
Hello! Just passing through and noticed that 1.3.1 is currently marked as the latest release rather than 1.3.2. Thought I'd point it out while I'm here since 1.3.1 is clearly marked as abandoned.

![image](https://user-images.githubusercontent.com/8540217/131161435-1a5e9aec-4857-4857-a3a7-bef909f676c4.png)

**Reply by jertel** [reply]
Fixed. Thanks.
